tmessage/db.py

```diff
@@ -1,28 +1,32 @@
 """Handles the connection to the SQLite database as well as DB interaction"""
 from datetime import datetime
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
 
-from peewee import CharField, DateTimeField, Model, SqliteDatabase
+app = Flask(__name__)
```

Comment:
There's no need for flask in tmessage because we are not making a webserver here. Let us know on which issue are you working on or try opening a new issue if you want to work on some other issues.

Thanks for the PR though :)

Reply:
I haven't used flask to make a webserver. I have used it so that I could use SqlAlchemy explicitly to work on this issue, and just because i was comfortable with SqlAlchemy ORM.

Reply:
You don't need flask to use sqlalchemy and secondly, we are using peewee as an ORM